docs/stencil-docs/installing-stencil-cli/stencil-cli-options-and-commands.md

```diff
@@ -114,9 +116,7 @@ stencil start --open # opens live theme preview in default browser
 | `--variation [<NAME>]`       |`-v` | Set which theme variation to use while developing                                     |
 | `--test`                     |`-t` | Enable QA mode which will bundle all javascript for speed to test locally             |
 | `--tunnel`                   |     | Create a tunnel URL which points to your local server which anyone can use            |
-| `--theme-editor`             |`-e` | Run Theme Editor server (**Deprecated as of v1.23.1**)                                                               |
```

Comment:
This has been removed from CLI now, so finally dropping from docs. The required version of CLI no longer has this.